cli/pcluster/utils.py

```diff
@@ -166,6 +166,14 @@ def get_instance_vcpus(region, instance_type):
     return vcpus
 
 
+def get_supported_os():
+    """
+    Return a tuple of the os supported by parallelcluster.
+    :return: a tuple of strings of the supported os
+    """
+    return "alinux", "centos6", "centos7", "ubuntu1404", "ubuntu1604"
```

Comment:
we can fixup this commit with the one on which you are using this function.
NOTE instead of a function we could have a global variable and use it directly.

Reply:
Instead we can have a `get_supported_os(scheduler)` function filtering the oses according to the selected scheduler